needle.pm

```diff
@@ -148,7 +148,7 @@ sub save {
         {
             tags       => [sort(@{$self->{tags}})],
             area       => \@area,
-            properties => [sort(@{$self->{properties}})],
+            properties => [$self->{properties}],
```

Comment:
@os-autoinst/tools-team I'm wondering whether we still want to have the properties sorted. A custom sort function which handles the hashes shouldn't be hard to implement.

Reply:
What's the problem with the old code?

Reply:
`$self->{properties}` now contains not only a list of strings but also references (to hashes). This must be taken into account when sorting. I don't know how `sort` handles references by default.

Reply:
`sort` on references is pointless, they just get stringified to their address.